python_modules/dagster/dagster/core/definitions/graph.py

```diff
@@ -182,6 +182,10 @@ def _solids_in_topological_order(self):
 
         return [self.solid_named(solid_name) for solid_name in order]
 
+    @property
+    def node_as_str(self) -> str:
```

Comment:
I think it would make sense to call this something like `node_type_str`.  `node_as_str` makes me thinkg it would return something like what `__repr__` returns.